CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/CameraPreviewViewController.swift

```diff
@@ -122,7 +122,6 @@ final class CameraPreviewViewController: UIViewController {
                                                name: NSNotification.Name.AVCaptureDeviceSubjectAreaDidChange,
                                                object: camera.videoDeviceInput?.device)
         view.backgroundColor = GiniColor(light: UIColor.GiniCapture.dark1, dark: UIColor.GiniCapture.dark1).uiColor()
-        previewView.drawGuides(withColor: giniConfiguration.cameraPreviewCornerGuidesColor)
```

Comment:
[Clarification] If we remove this Configuration we need to use a color from the `GiniColors` palette instead.